### PR TITLE
Revert "cmd/podman: add support for checkpoint images"

### PR DESCRIPTION
**Pull request by rhatdan**
This reverts commit 9c3d8bb494f358ecff785ba81a58f2e05f1a98a1.

This patch needs to be reworked to happen on the server side not on the client side.

Replaces https://github.com/containers/podman/pull/16068

#### Does this PR introduce a user-facing change?

```release-note
None
```
[NO NEW TESTS NEEDED]

### CODE REVIEW

**Comment by rhatdan** [comment]
@rst0git I think we need to revert this merge and rework it on the server side not on the client side.

**Review by rst0git — APPROVED** [review]
LGTM

**Comment by mheon** [comment]
Tests are very red

**Comment by mheon** [comment]
@rhatdan Github is complaining about a DCO?

**Review by vrothberg — COMMENTED** [review]
/lgtm

File: docs/source/markdown/podman-kube-play.1.md.in [hunk 1]
```diff
@@ -233,7 +233,7 @@ Pods removed:
 `podman kube play --down` will not work with a URL if the YAML file the URL points to
 has been changed or altered.
 
-
+@@option tls-verify
```
Comment (edsantiago) [inline docs/source/markdown/podman-kube-play.1.md.in:236]: @rhatdan do you have any idea why you added this here? It's weird and out of place, I'm 99.99% sure it's an oops. I am going to remove it in my upcoming include-mechanism PR.

Comment (rhatdan) [reply docs/source/markdown/podman-kube-play.1.md.in:236]: SGTM

**Pull request merged**